api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -107,4 +117,21 @@ public String getFreeTierBillingProjectName() {
   public void setFreeTierBillingProjectName(String freeTierBillingProjectName) {
     this.freeTierBillingProjectName = freeTierBillingProjectName;
   }
+
+  // Authorities (special permissions) are granted only through direct db edits, for example
+  // INSERT INTO authority VALUES(
+  //     (SELECT user_id FROM user WHERE email = 'me@staging.pmi-ops.org'),
+  //     0);
+  // TODO(RW-85) Tool to edit authorities.
+  // TODO Switch to FetchMode.JOIN when if we upgrade past Hibernate 3.
```

Comment:
Not sure if you saw my latest comments -- we don't want to do an eager join here since the data is only used in rare cases. (And if we did want to do an eager join we could still accomplish that by writing our own SQL.)

You can remove this TODO.